.github/workflows/ci_eval_short.yaml

```diff
@@ -74,4 +74,4 @@ jobs:
             iree-base-runtime
 
       - name: Run perplexity test with vmfb
-        run:  pytest -n 8 -v -s sharktank/tests/evaluate/perplexity_iree_test.py --run-quick-llama-test --bs=5 --iree-device='hip://6' --iree-hip-target=gfx942 --iree-hal-target-backends=rocm --llama3-8b-f16-model-path=/data/llama3.1/8b/llama8b_f16.irpa --llama3-8b-tokenizer-path=/data/llama3.1/8b/tokenizer_config.json
```

Comment:
Is there a reason we are targeting only device hip://6? Should we have the runners available decide the device used?

Reply:
Ideally, but the specific `iree_device` needs to be passed to the `vmfbRunner`. We might require another script/flag to determine a free device and pass that info dynamically. Let me know if it's worth looking into.

Reply:
@saienduri have we setup ways to target specific devices on runners outside hardcoding like this?

Reply:
Yeah we do. We can specify ROCR_VISIBLE devices so that runners initialize with only certain gpus to avoid conflicts. I can make sure the seperation is in place to avoid conflicts on Monday.